pkg/linter/linter.go

```diff
@@ -565,15 +571,76 @@ func (l Linter) lintWithRegoRules(
 	}
 }
 
+func (l Linter) lintWithRegoAggregateRules(
```

Comment:
Does it makes sense to cover this in linter_test too?

Reply:
I've added a simple test 👍 And as it's covered by an e2e test too it's probably good enough for the time being.